assets/queries/cloudFormation/rds_open_to_large_scope/query.rego

```diff
@@ -0,0 +1,66 @@
+package Cx
+
+CxPolicy [ result ] {
+	some i
+        not input.document[i].Resources[name].Type == "AWS::RDS::DBSecurityGroup"
```

Comment:
Why are you checking "not input.document[i].Resources[name].Type == "AWS::RDS::DBSecurityGroup" if bellow you check if resource.Type == "AWS::EC2::SecurityGroup"

Reply:
According to the documentation, when a DBSecurityGroup is defined, there must not be a VPCSecurityGroup, so that was how I was checking. But now that I think about it, checking that is not the goal of the query, so I am going to remove that line.

Reply:
According to the documentation, when a DBSecurityGroup is defined, there must not be a VPCSecurityGroup, so that was how I was checking. But now that I think about it, checking that is not the goal of the query, so I am going to remove that line.